packages/editor-ui/src/plugins/N8nCustomConnectorType.js

```diff
@@ -0,0 +1,470 @@
+			}
+			else {
+				p.push(point[1] + (majorAnchor * soo[1]));
+			}
+		}
+
+		return p;
+	};
+
+
+	function sgn(n) {
+		return n < 0 ? -1 : n === 0 ? 0 : 1;
+	};
+
+	function getFlowchartSegmentDirections(segment) {
+		return [
+			sgn( segment[2] - segment[0] ),
+			sgn( segment[3] - segment[1] ),
+		];
+	};
+
+	function getSegmentLength(s) {
+		return Math.sqrt(Math.pow(s[0] - s[2], 2) + Math.pow(s[1] - s[3], 2));
+	};
+
+	function _cloneArray(a) {
+		var _a = [];
+		_a.push.apply(_a, a);
+		return _a;
+	};
+
+	function writeFlowchartSegments(_super, conn, segments, paintInfo, cornerRadius) {
+		var current = null, next, currentDirection, nextDirection;
+		for (var i = 0; i < segments.length - 1; i++) {
+
+			current = current || _cloneArray(segments[i]);
+			next = _cloneArray(segments[i + 1]);
+
+			currentDirection = getFlowchartSegmentDirections(current);
+			nextDirection = getFlowchartSegmentDirections(next);
+
+			if (cornerRadius > 0 && current[4] !== next[4]) {
+
+				var minSegLength = Math.min(getSegmentLength(current), getSegmentLength(next));
+				var radiusToUse = Math.min(cornerRadius, minSegLength / 2);
+
+				current[2] -= currentDirection[0] * radiusToUse;
+				current[3] -= currentDirection[1] * radiusToUse;
+				next[0] += nextDirection[0] * radiusToUse;
+				next[1] += nextDirection[1] * radiusToUse;
+
+				var ac = (currentDirection[1] === nextDirection[0] && nextDirection[0] === 1) ||
+													((currentDirection[1] === nextDirection[0] && nextDirection[0] === 0) && currentDirection[0] !== nextDirection[1]) ||
+													(currentDirection[1] === nextDirection[0] && nextDirection[0] === -1),
+					sgny = next[1] > current[3] ? 1 : -1,
+					sgnx = next[0] > current[2] ? 1 : -1,
+					sgnEqual = sgny === sgnx,
+					cx = (sgnEqual && ac || (!sgnEqual && !ac)) ? next[0] : current[2],
+					cy = (sgnEqual && ac || (!sgnEqual && !ac)) ? current[3] : next[1];
+
+				_super.addSegment(conn, STRAIGHT, {
+					x1: current[0], y1: current[1], x2: current[2], y2: current[3],
+				});
+
+				_super.addSegment(conn, ARC, {
+					r: radiusToUse,
+					x1: current[2],
+					y1: current[3],
+					x2: next[0],
+					y2: next[1],
+					cx: cx,
+					cy: cy,
+					ac: ac,
+				});
+			}
+			else {
+				// dx + dy are used to adjust for line width.
+				var dx = (current[2] === current[0]) ? 0 : (current[2] > current[0]) ? (paintInfo.lw / 2) : -(paintInfo.lw / 2),
+					dy = (current[3] === current[1]) ? 0 : (current[3] > current[1]) ? (paintInfo.lw / 2) : -(paintInfo.lw / 2);
+
+				_super.addSegment(conn, STRAIGHT, {
+					x1: current[0] - dx, y1: current[1] - dy, x2: current[2] + dx, y2: current[3] + dy,
+				});
+			}
+			current = next;
+		}
+		if (next != null) {
+			// last segment
+			_super.addSegment(conn, STRAIGHT, {
+				x1: next[0], y1: next[1], x2: next[2], y2: next[3],
+			});
+		}
+	};
+
+	const lineCalculators = {
```

Comment:
I removed other types of flows... when endpoints are perpendicular or orthogonal... to avoid lots of dead code.. can add them back if needed